docs/clients.md

```diff
@@ -116,6 +116,12 @@ xmap <silent> <Leader>c             <Plug>(coc-codeaction-selected)
 nmap <silent> <Leader>c             <Plug>(coc-codeaction-line)
 nmap <silent> gd                    <Plug>(coc-definition)
 ```
+---
+
+## Nvim
```

Comment:
Could you make nvim a subsection of vim?
`###`

Reply:
:+1: